liqd_product/apps/partners/templates/partner_base.html

```diff
@@ -0,0 +1,38 @@
+{% extends 'base.html' %}
+
+{% load userindicator static i18n %}
+
+{% block after_header %}
+<img src="{{partner.logo}}" alt="Logo" />
```

Comment:
Uhhh, I think I fixed that in the nav PR. https://github.com/liqd/a4-product/pull/10/commits/17eee82c386252ad6d586361534d2eef60b620fa#diff-a7ef4f52ee1fdb9929f7d37d22da4d19R10